lms/djangoapps/ccx/tests/test_utils.py

```diff
@@ -62,7 +67,7 @@ def test_ccx_locator(self):
         self.assertEqual(result, ccx)
 
 
-class TestStaffOnCCX(CcxTestCase, SharedModuleStoreTestCase):
```

Comment:
because `CcxTestCase` already subclasses it:
https://github.com/mitocw/edx-platform/blob/master/lms/djangoapps/ccx/tests/utils.py#L26